docker-compose.yml

```diff
@@ -4,3 +4,5 @@ services:
   postgres:
     image: postgres:10
     ports: ["5432:5432"]
+    environment:
+        POSTGRES_DB: procrastinate
```

Comment:
(hm, when you say "for testing" in the commit name, it's about running automated tests, or trying out procrastinate manually ?)

Reply:
It's for trying out manually. With this, the postgres docker image automatically builds a `procrastinate` database when starting up. Otherwise, the demo app does not work out of the box.

I initially wanted to put it in another PR, trying to put most of the installation/manual testing in a Dockerfile and docker-compose. I guess I forgot when creating the PR. I can move this to another PR and rebase if you want, to keep the PRs well focused. 🙂

Reply:
It's totally ok to leave this in the PR. I'm just surprised that we never bumped into the issue before.

Reply:
I think that's because the README tells to export some shell variables prior to launch the docker-compose. And _maybe_ I did not follow the README…

I may try to look into simplifying that part in another PR. Imo, it would be nice if the first launch or manual testing would only require you to do `docker-compose up -d`. We could setup environment in docker-compose and put the demo app in its own container, automating all setup needed.

Reply:
yes, I'm totally for it ! (in another PR maybe, though)

So far I've tried to optimize the main use case without closing the door for users who want to use a postgres instance of their own and not the docker one. But if we can simplify that as much as possible, I'll always support that.